packages/console/src/components/ItemPreview/index.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.item {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+
+  > div:not(:first-child) {
+    margin-left: _.unit(4);
+  }
```

Comment:
This should not be applied inside the component. Ideally a basic component should not have any margins, and this helps make the components extra friendly to the callers. Otherwise the caller will probably have to write extra CSS code to override the unexpected margins.